docs/API_DESIGN.md

```diff
@@ -682,9 +685,85 @@ CLASS BearerDid
   PUBLIC DATA did: Did
   PUBLIC DATA document: Document
   CONSTRUCTOR(uri: string, key_manager: KeyManager)
+  CONSTRUCTOR(portable_did: PortableDid)
   METHOD get_signer(): Signer
 ```
 
 > [!WARNING]
 >
-> We'll need to add support for the developer to select a VM other than defaulting to the first in the `verification_method` array; add `METHOD get_signer_by_kid(key_id: string): Signer`.
\ No newline at end of file
+> We'll need to add support for the developer to select a VM other than defaulting to the first in the `verification_method` array; add `METHOD get_signer_by_kid(key_id: string): Signer`.
+
+### Example: Instantiate from a [`PortableDid`](#portabledid)
+
+```pseudocode!
+portable_did = JSON.parse(env.get("PORTABLE_DID"))
```

Comment:
we have a `CONSTRUCTOR(json: string)` for good reason (to give more useful error results), so we should use it here!